### ISSUE TITLE
Where can I get PGD0-A and PGD0-T?

### ISSUE THREAD

**Issue by tianhhhh** [CityU-MLO/sPGD#1]
Hi,
I couldn't find PGD0-A and PGD0-T. Where can I get them?

**Comment by XuyangZhong-29** [comment]
Hi,

Sorry for the late reply. We directly use the codes in https://github.com/fra31/sparse-imperceivable-attacks/tree/master to implement PGD0. The sole difference between PGD0-A and PGD0-T is the loss function we use during training. A: vanilla adversarial training loss, T: TRADES loss.

Hope this can address your issue.
